metadata/pkg/utils/ObjectMapper.go

```diff
@@ -0,0 +1,67 @@
+package utils
+
+import (
+	"github.com/opensds/multi-cloud/metadata/pkg/model"
+	"github.com/opensds/multi-cloud/metadata/proto"
+
+	pb "github.com/opensds/multi-cloud/metadata/proto"
+)
+
+func GetProtoBackends(unPaginatedResult []*model.MetaBackend) []*pb.BackendMetadata {
```

Comment:
please rename to simply, GetBackends()

Reply:
![image](https://user-images.githubusercontent.com/48655639/214242470-cab2ff99-0d48-41a2-98d4-db5c151f0278.png)
done